smack-integration-test/src/main/java/org/jivesoftware/smackx/muc/MultiUserChatIntegrationTest.java

```diff
@@ -161,4 +681,38 @@ public void roomDestroyed(MultiUserChat alternateMUC, String reason) {
         assertEquals(0, muc.getOccupantsCount());
         assertNull(muc.getNickname());
     }
+
+    /**
+     * Gets a random room name
+     *
+     * @param prefix A prefix to add to the room name for descriptive purposes
+     */
+    private EntityBareJid getRandomRoom(String prefix) throws XmppStringprepException {
+        final String roomNameLocal = String.join("-", prefix, testRunId, StringUtils.insecureRandomString(6));
+        return JidCreate.entityBareFrom(Localpart.from(roomNameLocal), mucService.getDomain());
+    }
+
+    /**
+     * Destroys a MUC room, ignoring any exceptions.
+     *
+     * @param muc The room to destroy (can be null).
+     */
+    static void tryDestroy(final MultiUserChat muc) {
+        if (muc == null) {
+            return;
+        }
+        try {
+            muc.destroy("test fixture teardown", null);
+        } catch (Exception ignore) {
+            LOGGER.log(Level.FINEST, "Failed to tear down a randomly generated MUC (" + muc.getRoom().toString() + ") used as a test fixture. Continuing anyway.");
```

Comment:
Always pass the exception to the logging subsystem. And please don't call the `Exception` variable `ignore`.

LOGGER.log(Level.FINEST, "Failed to tear down a randomly generated MUC (" + muc.getRoom().toString() + ") used as a test fixture. Continuing anyway.", exception);

Reply:
First exception has been wrapped. Second exception passed to logging. Thanks for the tips!